examples/platform/linux/ControllerShellCommands.cpp

```diff
@@ -165,6 +271,25 @@ static CHIP_ERROR DiscoverHandler(int argc, char ** argv)
     {
         return error = display(true);
     }
+    else if (strcmp(argv[0], "onnetwork") == 0)
```

Comment:
Consider using the argument parser like the one in `examples/platform/linux/Options.cpp` for safer argument processing.

Went through other commands, seems this is the first command with more complex arguments.